core/src/main/java/org/axonframework/config/SagaConfiguration.java

```diff
@@ -58,6 +62,16 @@
     private final List<Function<Configuration, MessageHandlerInterceptor<? super EventMessage<?>>>> handlerInterceptors = new ArrayList<>();
     private Configuration config;
 
+    @Override
+    public void start() {
+        // nothing to be started
```

Comment:
Why not, who's starting the sagas?

Reply:
Starting a saga means starting an event processor responsible for it. So, `EventProcessorRegistry` is responsible for starting it.

Reply:
Ah of course, all that stuff's been moved to the `EventProcessorRegistry`, makes sense now!